projects/iio_demo/src.mk

```diff
@@ -1,9 +1,23 @@
-#See No-OS/tool/scripts/src_model.mk for variable description
-
+# See No-OS/tool/scripts/src_model.mk for variable description
 SRC_DIRS += $(PROJECT)/src/app
 SRC_DIRS += $(NO-OS)/iio/iio_demo
 SRC_DIRS += $(NO-OS)/iio/iio_app
 
+LIBRARIES += iio
```

Comment:
I think this line is now redundant since the makefiles now only use the TINYIIOD define to determine if compiling the iio library is necessary. @mchindri can you confirm this?

Reply:
yes, LIBRARIES += iio is not needed anymore. TINYIIOD=y is enough

Reply:
removed it in latest version.